<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataExportForm</class>
 <widget class="QDialog" name="DataExportForm">
  <property name="windowTitle">
   <string>Экспорт проекта / данных</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelExportType">
     <property name="text"><string>Тип экспорта:</string></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="exportType">
     <item><property name="text"><string>Полный архив (ZIP)</string></property></item>
     <item><property name="text"><string>GeoJSON</string></property></item>
     <item><property name="text"><string>PDF/Word</string></property></item>
     <item><property name="text"><string>База SQLite/PostgreSQL</string></property></item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTarget">
     <property name="text"><string>Путь для сохранения:</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="targetPath"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonExport">
     <property name="text"><string>Экспортировать</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>